ghcide/src/Development/IDE/Types/HscEnvEq.hs

```diff
@@ -108,9 +124,9 @@ instance Eq HscEnvEq where
   a == b = envUnique a == envUnique b
 
 instance NFData HscEnvEq where
-  rnf (HscEnvEq a b c d _) =
+  rnf (HscEnvEq a b c d _ f) =
       -- deliberately skip the package exports map
-      rnf (hashUnique a) `seq` b `seq` c `seq` rnf d
+      rnf (hashUnique a) `seq` b `seq` c `seq` d `seq` rnf f
```

Comment:
If you make the field lazy you will want to exclude it from the `rnf` definition since Shake will call `rnf` every time that the rule is evaluated